thing-management.md

```diff
@@ -225,15 +231,26 @@ Message (command):
 }
 ```
 
-### Report for NPL Entry
+### Remove all NPL entries.
 
-The following report can occur in five situations:
+Topic: `pt:j1/mt:cmd/rt:ad/rn:zw/ad:1`
 
-1. After adding an entry (status="added").
-2. After deleting an entry (status="deleted" or status="deleted_only_from_npl").
-3. After editing an entry (status="edited", network_status: {"status": "not_in_network"}).
-4. After powering up a device (status="edited", network_status: {"status": "included"}).
-5. After inclusion in wrong network (status="included_bad_network").
+Message (command):
```

Comment:
FIMP message:

Reply:
That's how it's done everywhere in FIMP Docu, so You would like to change that, then another PR probably.